datafusion-cli/src/print_options.rs

```diff
@@ -110,7 +90,7 @@ impl PrintOptions {
         self.format
```

Comment:
One thing I noticed when reviewing this code is that there are different passes for print_batches and `print_table_batch`

I wonder if there is some way to combine them (so we always use the same printing logic)